test/wallaby/driver/process_workspace_test.exs

```diff
@@ -14,6 +14,24 @@ defmodule Wallaby.Driver.ProcessWorkspaceTest do
     def init([]), do: {:ok, []}
   end
 
+  defp now_in_milliseconds(), do: DateTime.utc_now() |> DateTime.to_unix(:millisecond)
+
+  defp with_timeout(doer, timeout \\ 100),
+    do: with_timeout(doer, now_in_milliseconds(), timeout)
+
+  defp with_timeout(doer, start, timeout) do
+    doer.()
+  rescue
+    e ->
+      passed_timeout? = now_in_milliseconds() - start >= timeout
+
+      if passed_timeout? do
+        reraise e, __STACKTRACE__
+      else
+        with_timeout(doer, start, timeout)
+      end
+  end
+
```

Comment:
What do you think about moving these private functions to the bottom of the module? When I first open up the file, it's a lot to digest when I'm really just curious in what the tests are.

Reply:
Sounds good 👍